supervise.py

```diff
@@ -329,3 +340,10 @@ def status(self):
         seconds = 0 if now < seconds else (now - seconds)
 
         return ServiceStatus(status=status, pid=pid, action=action, uptime=seconds)
+
+
+class ServiceStatusError(Exception):
```

Comment:
Add a shared 'SuperviseError' base class. s/Service/Supervise/ for consistency with the module name.